packages/roosterjs-react-emoji/lib/components/EmojiNavBar.tsx

```diff
@@ -11,24 +11,29 @@ export interface EmojiNavBarProps {
     onClick?: (selected: string) => void;
     currentSelected?: string;
     getTabId?: (selected: string) => string;
+    navbarThemeClassName?: string;
+    navbarButtonThemeClassName?: string,
```

Comment:
- exclude "navbar" from these class names, it's implied from the component itself
- exclude "theme", this is supposed to be agnostic of theme